frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import messageIcon from '../assets/messageIcon.svg'
+import adminIcon from '../assets/adminIcon.svg'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { RootState } from '../store/types'
+import { PATHS } from '../routes/Routes'
+import { DOMAIN_URL, ENDPOINTS } from '../store/endpoints'
+import { months, days } from '../common/dates'
+
+const BookingGroup = styled.div``
+const BookingContainer = styled.div`
+  position: relative;
+  cursor: pointer;
+  background-color: #ffffff;
+  margin: 23px;
+  min-height: 70px;
+  border-radius: 20px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+  padding: 15px;
+`
+const BookingLeftDisplay = styled.div`
+  align-self: center;
+  width: 80%;
+`
+
+const DateLabel = styled.div`
+  margin-bottom: 0em;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  overflow: hidden;
+  color: #ff705c;
+`
+
+const TimeLabel = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 14px;
+`
+const BookingCCAName = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 14px;
+`
+
+const BookingRightDisplay = styled.div`
+  position: absolute;
+  right: 15px;
+  top: 50%;
+  transform: translate(-17%, -50%);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+const Icon = styled.img`
+  padding: 20px;
+`
+export default function BookingCard() {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const params = useParams<{ facilityID: string }>()
+  const {
+    ViewStartDate,
+    ViewEndDate,
+    createSuccess,
+    createFailure,
+    isLoading,
+    facilityBookings,
+    selectedFacilityName,
+    selectedFacilityId,
+  } = useSelector((state: RootState) => state.facilityBooking)
+  const getHumanReadableTime = (eventStartTime: number) => {
+    const date = new Date(eventStartTime * 1000)
+    let hour = date.getHours().toString()
+    if (hour.length == 1) {
+      hour = '0' + hour
+    }
+    let minutes = date.getMinutes().toString()
+    if (minutes.length == 1) {
+      minutes = '0' + minutes
+    }
+
+    return hour + minutes
+  }
+
+  const getHumanReadableDate = (eventTime: number) => {
```

Comment:
consider abstracting this into the `/src/common` folder, follow unixTo12HourTime format, including a comment on how the output will look like too

Reply:
Resolved in commit 9c59d802492bb5f95b587c34ae11e07f5965c73a. Abstracted into common folder, not sure about naming.